packages/noya-designsystem/src/components/Button.tsx

```diff
@@ -11,7 +11,8 @@ type ButtonVariant = 'normal' | 'thin';
 const ButtonElement = styled.button<{
   active: boolean;
   variant: ButtonVariant;
-}>(({ theme, active, disabled, variant }) => ({
+  full: boolean;
```

Comment:
We might need to add this, but if we do, I think we should try to use flexbox.

In this case though, I think we can get the right layout by making `PaddedSection` a flex container:

![Screen Shot 2021-06-03 at 11 08 23 AM](https://user-images.githubusercontent.com/1198882/120692014-0d0e2c00-c45c-11eb-96a7-be745533af4b.png)